jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -213,18 +215,39 @@ public static String getSkaffoldSyncMap(
             .getLayerEntries()
             .stream()
             .filter(layerEntry -> Files.isRegularFile(layerEntry.getSourceFile()))
+            .filter(
+                layerEntry ->
+                    !excludesExpanded.contains(layerEntry.getSourceFile().toAbsolutePath()))
```

Comment:
Hmm, I'm wondering if this works with paths where the project ancestry has symlinks.

Reply:
interesting... I'll check it out.

Reply:
might have to `isSameFile` it?

Reply:
Ok both ok since they seem to expand the project path.